ide/projectui/src/org/netbeans/modules/project/ui/resources/license-default.txt

```diff
@@ -1,4 +1,4 @@
 ${licenseFirst!""}
-${licensePrefix}Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
-${licensePrefix}Click nbfs://nbhost/SystemFileSystem/${.data_model["org.openide.filesystems.FileObject"].getPath()} to edit this template
+${licensePrefix}Ctrl/Cmd+Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+${licensePrefix}Ctrl/Cmd+Click nbfs://nbhost/SystemFileSystem/${.data_model["org.openide.filesystems.FileObject"].getPath()} to edit this template
```

Comment:
`Ctrl/Cmd` sounds horrible. I don't have Cmd on my keyboard! Would `See nbfs:/....` work?

Reply:
the problem was that many didn't know how to click the link as seen in issues and on the mailing list. I don't have Cmd either. If I wouldn't have added it someone would have commented that they had no Ctrl on their kb.

Reply:
I have noticed the complaint on mailing list as well. Yes, we have a usability problem. 

> I don't have Cmd either. If I wouldn't have added it someone would have commented that they had no Ctrl on their kb.

Probably a sign that the user experience deserves deeper fix than `Ctrl/Cmd` prefix.

Reply:
It could be a sign. But maybe its sufficient to explain how the link works at least until someone is interested to implement a deeper fix.

Reply:
I'd like to point out that users not knowing how to "click" in the editor (e.g. with Ctrl or Cmd) probably don't know how to navigate to symbol definitions either!  _"Addressing"_ just the URL problem without fixing the underlying usability issue makes little sense to me.

I am fine with some rewording - for example `Visit nbfs:...` or `Navigate to nbfs:` - and then letting the user intelligence work - but I find `Ctrl/Cmd+Click` so ugly, that I cannot adore it... even if it helped even a single user.

Reply:
+1 to `Visit nbfs:...`, and possibly thinking about better ways of guiding the user into how to click things separately.